app/models/stripe_wrapper.rb

```diff
@@ -1,36 +1,63 @@
-module StripeWrapper
-  class Charge
-    attr_reader :status, :response
+class StripeWrapper
+  attr_reader :status, :response
+  delegate :id, to: :@response
+  delegate :message, to: :@response, prefix: :error
 
-    def initialize(response, status)
-      @response = response
-      @status = status
-    end
+  def initialize(response, status)
+    @response = response
+    @status = status
+  end
+
+  def successful?
+    status == :success
+  end
+
+  def self.set_api_key
+    Stripe.api_key = ENV.fetch('STRIPE_API_KEY')
+  end
 
+  class Customer < StripeWrapper
```

Comment:
Gotcha, makes sense. Since `Card` and `Customer` are indeed wrappers...I'm okay with that.